PATCH-P0-FIX-SUSE: set libdir for aarch64 correctly

Index: cmake/install_layout.cmake
===================================================================
--- cmake/install_layout.cmake
+++ cmake/install_layout.cmake
@@ -140,7 +140,7 @@ SET(INSTALL_SBINDIR_RPM
 SET(INSTALL_SCRIPTDIR_RPM               "bin")
 SET(INSTALL_SYSCONFDIR_RPM		"/etc")
 #
-IF(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64" "OR" CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64")
+IF(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64" "OR" CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64" OR CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
   SET(INSTALL_LIBDIR_RPM                "lib64")
   SET(INSTALL_PLUGINDIR_RPM             "lib64/mysql/plugin")
 ELSE()
