--- mysql.spec.in.orig	2014-05-23 13:25:00.500246270 +0200
+++ mysql.spec.in	2014-05-23 13:50:47.696903650 +0200
@@ -49,17 +49,13 @@
 Source3:        README.debug
 Source4:        suse-test-run
 Source5:        mysql.SuSEfirewall2
-Source7:        README.install
+Source7:        rc.mysql-multi
 Source8:        README.SUSE
 Source9:        mysql-patches.tar.bz2
 Source10:       build.inc
 Source11:       install.inc
 Source12:       series
 Source13:       configuration-tweaks.tar.bz2
-Source14:       my.ini
-Source15:       rc.mysql.sysvinit
-Source16:       rc.mysql.systemd
-Source17:       mysql.service
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 %if 0%{?suse_version}
 PreReq:         pwdutils
