--- rc.mysql-multi.body	2013-08-02 12:51:29.684015559 +0200
+++ rc.mysql-multi.body	2013-08-03 11:06:49.375189637 +0200
@@ -335,21 +335,6 @@
 		# necessary, but doesn't do any harm. On big upgrade, require 
 		# user confirmation.
 		if [ -f "/var/lib/mysql/.run-mysql_upgrade" ]; then
-			if [ \! -f "/var/lib/mysql/.force_upgrade" ] && \
-			   [ -f "$datadir/mysql_upgrade_info" ] && \
-			   [ -z "`grep "^$MYSQLVER" "$datadir/mysql_upgrade_info" 2> /dev/null`" ]; then
-			   echo
-			   echo "You are upgrading from different stable version of MySQL!"
-			   echo "Make sure you to make backup of your data (mostly $datadir)!"
-			   echo
-			   echo "If you want to continue upgrading your database, run following commands:"
-			   echo
-			   echo "   touch /var/lib/mysql/.force_upgrade"
-			   echo "   rcmysql restart"
-			   echo
-			   rc_failed;  rc_status -v; rc_exit
-
-			fi
 			echo > "$log_upgrade"
 			echo "`LANG="" date` - upgrading MySQL..." >> "$log_upgrade"
 			echo >> "$log_upgrade"
