Index: mysql.spec.in
===================================================================
--- mysql.spec.in	(revision 7ab8d4309361022ac72d2ac999f56760)
+++ mysql.spec.in	(working copy)
@@ -472,17 +472,6 @@
 	rmdir %{_rundir}/mysql/restart || :
 	rmdir %{_rundir}/mysql         || :
 fi
-# warn on first run
-datadir="/var/lib/mysql"
-if [ -x /usr/bin/my_print_defaults ]; then
-	datadir="`/usr/bin/my_print_defaults mysqld mysql_server | sed -n 's|--datadir=||p'`"
-	[ "$datadir" ] || datadir="/var/lib/mysql"
-fi
-if [ \! -d "$datadir/mysql" ]; then
-cat > /var/adm/update-messages/%{name}-%{version}-%{release} << EOF
-%(cat %_sourcedir/README.install)
-EOF
-fi
 
 #######################################################################
 # Various ldconfig post scripts                                       #
